templates/all-services/index.json-ld

```diff
@@ -118,9 +127,25 @@
 	],
 	"changesets": [
 		{
-			"@id": "_:cs_allservices",
+			"@id": "_:cs_allservices_2",
 			"name": "All services page",
 			"status": "stable",
+			"baseOnIteration": "_:iteration_allservices_3",
+			"detectableBy": "By the sequence of elements.",
+			"semantic": "h1 + h2*3 + h1 + h3*n + h2 + h3*3",
+			"context": "This page template is strictly limited to the All services page located on Canada.ca.",
```

Comment:
Please add back the accessibility notice from the other changeset since were keeping the double H1.